### DISCUSSION TITLE
Feature Request: Blockquote Button

### DISCUSSION THREAD

**Discussion by mbbischoff** [community/community#12] [Mobile] [answered]
The Markdown editor in the mobile app currently lacks a button to blockquote. This is very useful when having complex conversations abs when pasting in emails from customer support. I’d suggest adding it and having it add a < character or if text is selected, transform the selection into a Markdown blockquote.

![9D734D76-90B3-41EE-A441-02563B911C83](https://user-images.githubusercontent.com/14093/96304014-66a71080-0fc9-11eb-96ac-62c5c6064dac.png)

**Comment by rnystrom** [comment]
I've also found myself wanting a block quote control! [Linking to my comment](https://github.com/github/feedback/discussions/17#discussioncomment-109339) on checkbox controls and how to scale the UX. I'd love to ask you the same question:

> What do you think of Apple's "Notes" app design for controls? Are there any other writing tools you think have great style controls?

**Reply by mbbischoff** [reply]
Responded there.

**Comment by rnystrom** [comment, marked as answer]
Shipped today in 1.3.3 on iOS!
